backend/node/genvm/base.py

```diff
@@ -492,8 +511,9 @@ def get_contract_data(
                         EventScope.GENVM,
                         "Call method: " + method_name,
                         {
-                            "method_name": method_name,
-                            "method_args": method_args,
+                            "calldata": str(
```

Comment:
@kp2pml30 this is for logging purposes in the Simulator's frontend's UI. Can't we send the method name and args as before? is it because of their serialization?

Reply:
Hmmm. There is an issue that you can't `json.dumps` them, but in general it is possible to convert it to a string (with both f string or writing that serialization)

Reply:
Implemented string representation in python and now it is dumped

Example:
```
genlayer-simulator-jsonrpc-1             | SUCCESS  | [GenVM] Deploying contract {"calldata": "{'args':[],'method':'__init__',}", "output": "Hello! How can I assist you today?\n"}
```